hearthstone/battlebots/cheapo_bot.py

```diff
@@ -35,9 +35,6 @@ async def buy_phase_action(self, player: 'Player') -> StandardAction:
         return EndPhaseAction(FreezeDecision.NO_FREEZE)
 
     async def discover_choice_action(self, player: 'Player') -> DiscoverChoiceAction:
-        discover_cards = player.discover_queue[0]
+        discover_cards = player.discover_queue[0].items
         discover_cards = sorted(discover_cards, key=lambda card: card.tier)
```

Comment:
This is going to cause problems, since if you are discovering something other than a card, the tier won't be present and the code will crash.